docs/excel/show-hide-add-in.md

```diff
@@ -0,0 +1,77 @@
+
+> [!IMPORTANT]
+> The features described in this article are currently in preview and subject to change. They are not currently supported for use in production environments. To try the preview features, you will need to [join Office Insider](https://insider.office.com/join).
+> A good way to try out preview features is by using an Office 365 subscription. If you don't already have an Office 365 subscription, you can get one by joining the [Office 365 Developer Program](https://developer.microsoft.com/office/dev-program).
+
+## Configure an add-in to use a shared runtime
+
+To configure the add-in to use a shared runtime, see [Configure your Office Add-in to use a shared runtime](configure-your-add-in-to-use-a-shared-runtime.md).
+
+## Show and hide the task pane
+
+The new APIs are in the `Office.addin` namespace. To show the task pane, your code calls `Office.addin.showAsTaskpane()`. Office will display in a task pane the page that you assigned to the resource ID (`resid`) for the task pane. This is the `resid` that you assigned to the `<SourceLocation>` of the `<Action xsi:type="ShowTaskpane">` in the manifest. (See [Configure your Office Add-in to use a shared runtime](configure-your-add-in-to-use-a-shared-runtime.md).)
+
+This is an asynchronous method, so code that should not run until it completes should be awaited, either with the `await` keyword or with a `then()` method, depending on which JavaScript syntax you are using. The following assumes that there is an Excel worksheet named **CurrentQuarterSales**. The task pane should be made visible whenever this worksheet is activated. The method `onCurrentQuarter` is a handler for the [Office.Worksheet.onActivated](/javascript/api/excel/excel.worksheet?view=excel-js-preview#onactivated) event which has been registered for the worksheet.
+
+```javascript
+function onCurrentQuarter() {
+    Office.addin.showAsTaskpane()
+    .then(function() {
+        // Code that enables task pane UI elements for
+        // working with the current quarter.
+    });
+}
+```
+
+To hide the task pane, your code calls `Office.addin.hide()`. The following example is a handler that is registered for the [Office.Worksheet.onDeactivated](/javascript/api/excel/excel.worksheet?view=excel-js-preview#ondeactivated) event.
+
+```javascript
+function onCurrentQuarterDeactivated() {
+    Office.addin.hide();
+}
+```
+
+### Preservation of state and event listeners
+
+The `hide()` and `showAsTaskpane()` methods only change the *visibility* of the task pane. They do not unload or reload it (or reinitialize its state).
+
+Consider the following scenario: A task pane is designed with tabs. The **Home** tab is open when the add-in is first launched. Suppose a user opens the **Settings** tab and, later, code in the task pane calls `hide()` in response to some event. Still later code calls `showAsTaskpane()` in response to another event. The task pane will reappear, and the **Settings** tab is still elected.
```

Comment:
Is there a way to illustrate the two scenarios through pictures? Or simplify the written descriptions? I get what they are saying but took me a couple times of moving my hands around to simulate the tabs to figure out what was going on. Not sure if this is specific to me as a physical/visual learner. I'm gonna also tag @AlexJerabek to see what he thinks on this one as I am dealing with some cognitive effects from a condition I have today and unsure if that is coming into play in my difficulty figuring this out quickly.

Reply:
I almost don't know if you need the example given you say that it doesn't reload or change state in this first one

Reply:
Good catch. There was a graphic in the private preview Word doc, but I seem to have lost it when converting it to markdown. I've put it back in.